### PR TITLE
Fix email addresses not being add when user is created

### PR DESCRIPTION
**Pull request by epicserve**

### CODE REVIEW

**Comment by epicserve** [comment]
This PR fixes #206. I'm happy to make edits or go in a different direction.

**Comment by epicserve** [comment]
@jefftriplett or @frankwiles, just wondering if either of you will have bandwidth to review and move this forward. There is also another bug caused by the most resent release that has PR ready.

File: test_plus/test.py [hunk 1]
```diff
@@ -266,7 +266,11 @@ def make_user(cls, username='testuser', password='password', perms=None):
             user_factory = User.objects.create_user
 
         USERNAME_FIELD = getattr(User, 'USERNAME_FIELD', 'username')
-        test_user = user_factory(**{USERNAME_FIELD: username})
+        EMAIL_FIELD = getattr(User, 'EMAIL_FIELD', None)
+        user_data = {USERNAME_FIELD: username}
```
Comment (jefftriplett) [inline test_plus/test.py:270]: Let's switch these two lines to keep the logic grouping together based on the variable names:

```suggestion
        user_data = {USERNAME_FIELD: username}
        EMAIL_FIELD = getattr(User, 'EMAIL_FIELD', None)
```

**Review by jefftriplett — CHANGES_REQUESTED** [review]

**Comment by epicserve** [comment]
@jefftriplett done ✅

**Review by jefftriplett — APPROVED** [review]
🎉  Looks good to me. I'll defer the merge and final judgement to @frankwiles because I don't have deploy access.

**Comment by frankwiles** [comment]
Thanks @epicserve doing a release right now.

**Pull request merged**